### ISSUE TITLE
[Bug] HedgeModManager Crash With Keep Open Disabled

### ISSUE THREAD

**Issue by Al-AmeenAdewunmi** [thesupersonic16/HedgeModManager#130]
HMM Info:
```
    Version: 7.2-5
    Args: 
    StartDir: C:\Program Files (x86)\Steam\steamapps\common\SonicForces\build\main\projects\exec
    Process Level: Administrator
    Game: Sonic Forces
    SteamGame: Sonic Forces
```

Exception:
```
    Type: InvalidOperationException
    Message: The calling thread cannot access this object because a different thread owns it.
    Source: WindowsBase
    Function: Void VerifyAccess()
    StackTrace: 
       at System.Windows.Threading.Dispatcher.VerifyAccess()
       at System.Windows.Application.CriticalShutdown(Int32 exitCode)
       at HedgeModManager.MainWindow.StartGame()
       at HedgeModManager.MainWindow.<<UI_SaveAndPlay_Click>b__27_0>d.MoveNext()
    InnerException: 
```
[HMM_Snapshot_00082020081902.txt](https://github.com/thesupersonic16/HedgeModManager/files/5067661/HMM_Snapshot_00082020081902.txt)

**Comment by thesupersonic16** [comment]
Thanks for posting this issue. The issue seems to be related to closing HMM after you press Play.
For the time being in your settings Enable `Keep Hedge Mod Manager open after starting a game`

**Comment by thesupersonic16** [comment]
Fixed in 40c458aedf4bf7dac80e2d352c0df85801b944b6. This issue will be closed on the next release.
